v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password + Social Login",
```

Comment:
The hover logic does not account for the outer container it seems


https://user-images.githubusercontent.com/18233774/225236745-634aa403-31a9-4984-82ae-4c0595e1bb9b.mov